gwcs/wcstools.py

```diff
@@ -302,7 +302,7 @@ def wcs_from_points(xy, world_coords, proj_point='center',
                          "Only one of {} is supported.".format(polynomial_type,
                                                                supported_poly_types.keys()))
 
-    skyrot = models.RotateCelestial2Native(crval[0], crval[1], 180*u.deg)
+    skyrot = models.RotateCelestial2Native(crval[0].to_value(u.deg), crval[1].to_value(u.deg), 180)
```

Comment:
This was causing the generated transform to have a mix of using Quantity and not using it, this makes it all not-quantity.